web_src/less/index.less

```diff
@@ -1,5 +1,7 @@
 @import "~font-awesome/css/font-awesome.css";
 @import "./vendor/gitGraph.css";
+@import "./features/animations.less";
+@import "./markdown/mermaid.less";
```

Comment:
why not "./features/mermaid.less" ? There's nothing restricting mermaid to markdown...

Reply:
Right now it only appears in markdown and this file naming mirrors the one in `js`.

Reply:
Thought I guess if you prefer, we can move both to `features`, but standalone mermaid template files are not possible currently.

Reply:
Actually, I'm not going to change this. It's a markdown-only feature. Also in line with [gitlab's](https://gitlab.com/gitlab-org/gitlab/-/blob/master/app/assets/javascripts/behaviors/markdown/render_mermaid.js) file naming scheme if that matters.